confluence/server/scio_search/src/main/java/com/askscio/atlassian_plugins/confluence/impl/ScioSearchInfoResponse.java

```diff
@@ -0,0 +1,126 @@
+    }
+  }
+
+  @JsonAutoDetect(fieldVisibility = Visibility.ANY)
+  public static class InstanceInfo {
+
+    private final String version;
+    private final String baseUrl;
+    private final List<InstalledPluginInfo> installedPluginInfos;
+
+    public InstanceInfo(SettingsManager settingsManager, PluginAccessor pluginAccessor,
+        UserManager userManager,
+        boolean getInstalledPlugins) {
+      this.version = GeneralUtil.getVersionNumber();
+      this.baseUrl = settingsManager.getGlobalSettings().getBaseUrl();
+      if (getInstalledPlugins && Utils.isCurrentUserAdmin(userManager)) {
+        this.installedPluginInfos = new ArrayList<>();
+        Collection<Plugin> installedPlugins = pluginAccessor.getPlugins();
+        for (Plugin plugin : installedPlugins) {
+          InstalledPluginInfo installedPluginInfo = new InstalledPluginInfo(pluginAccessor, plugin);
+          this.installedPluginInfos.add(installedPluginInfo);
+        }
+      } else {
+        this.installedPluginInfos = null;
+      }
+    }
+
+    @JsonAutoDetect(fieldVisibility = Visibility.ANY)
+    public static class InstalledPluginInfo {
+
+      private final String name;
+      private final String key;
+      private final String description;
+      private final String vendorName;
+      private final String vendorUrl;
+      private final String version;
+      private final boolean isEnabled;
+
+      public InstalledPluginInfo(PluginAccessor accessor, Plugin plugin) {
+        PluginInformation pluginInformation = plugin.getPluginInformation();
+        this.name = plugin.getName();
+        this.key = plugin.getKey();
+        this.description = pluginInformation.getDescription();
+        this.vendorName = pluginInformation.getVendorName();
+        this.vendorUrl = pluginInformation.getVendorUrl();
+        this.version = pluginInformation.getVersion();
+        this.isEnabled = accessor.isPluginEnabled(plugin.getKey());
+      }
+    }
+  }
+
+  @JsonAutoDetect(fieldVisibility = Visibility.ANY)
+  public static class ScioPluginInfo {
+
+    private final String version;
+    private final String target;
+    private final PluginStatus pluginStatus;
+
+    public ScioPluginInfo(PluginAccessor pluginAccessor,
+        PluginSettingsFactory pluginSettingsFactory) {
+      final PluginSettings pluginSettings = pluginSettingsFactory.createGlobalSettings();
+      final Plugin plugin = pluginAccessor.getPlugin(Constants.PLUGIN_KEY);
+      if (plugin != null && plugin.getPluginInformation() != null) {
+        this.version = pluginAccessor.getPlugin(Constants.PLUGIN_KEY).getPluginInformation()
+            .getVersion();
+      } else {
+        logger.warn(String.format("Plugin version not found for %s", Constants.PLUGIN_KEY));
+        this.version = null;
+      }
+      this.target = (String) pluginSettings.get(TARGET_CONFIG_KEY);
+      this.pluginStatus = Utils.getPluginStatus(pluginSettings);
```

Comment:
I'm sure there's a more direct way to read this from the pom, but I don't know it offhand.

Reply:
i didn't get this part -- getPluginStatus returns info regarding webhook POST events such as latest response code, latest success time, latest failure time and so on. We will not be able to get this from the pom right?